features/admin.extensions.v1/components/logs/pages/logs.tsx

```diff
@@ -129,9 +130,17 @@ const LogsPage: FunctionComponent<LogsPageInterface> = (
             <PageLayout
                 title={ t("extensions:develop.monitor.pageHeader.title") }
                 pageTitle="Logs"
-                description={
-                    t("extensions:develop.monitor.pageHeader.description")
-                }
+                description={ (
+                    <>
+                        { t("extensions:develop.monitor.pageHeader.description") }
+                        <DocumentationLink
+                            link="manage.logs.learnMore"
```

Comment:
Need to use `getLink` when retrieving the link.

Reply:
https://github.com/wso2/identity-apps/blob/master/features/admin.applications.v1/pages/applications.tsx#L621